### DISCUSSION TITLE
Fetch Error on localhost and on server refresh cause the providers to be null

### DISCUSSION THREAD

**Discussion by muftisamiullah** [nextauthjs/next-auth#2569] [Help]
### Question 💬

logger.js?3836:11 [next-auth][error][client_fetch_error] 
https://next-auth.js.org/errors#client_fetch_error session SyntaxError: Unexpected token < in JSON at position 0

POST http://localhost:3000/api/auth/_log 405 (Not Allowed)

next.config.js contains:

        basePath:'http://localhost:3000', 
        HOST:`http://localhost:8000`,        
        NEXTAUTH_URL:`http://localhost:3000`,

.env.local contains:

      HOSTNAME=127.0.0.1
      PORT=3000
      HOST=http://$HOSTNAME:$PORT
      NEXTAUTH_URL=http://localhost:3000

### How to reproduce ☕️

running on localhost i dont know why im getting this.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by muftisamiullah** [comment]
i was able to fix the error by deleting the .next build folder and creating a new build, the only thing i didnt undersatnd why was it taking from build folder

**Reply by ceres-quangd** [reply]
I'm getting same problem

**Reply by muftisamiullah** [reply]
try the above
